components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LocalEntryAdminClient.java

```diff
@@ -113,7 +114,7 @@ public void addLocalEntry(String content) throws AxisFault {
      */
     public Boolean getEntry(String key) throws AxisFault {
         try {
-            Object object = localEntryAdminServiceStub.getEntry(key);
+            Object object = localEntryAdminServiceStub.getEntry(key, tenantDomain);
```

Comment:
can you name this variable appropriately?

Reply:
Updated